README.md

```diff
@@ -28,7 +28,7 @@ It is written with an aim to fasten golang setup.
 ```sh
 # wget or curl the script
 $ wget https://raw.githubusercontent.com/solodynamo/go-init/master/goinit.sh
-$ curl https://raw.githubusercontent.com/solodynamo/go-init/master/goinit.sh
+$ curl -o ~/goinit.sh https://raw.githubusercontent.com/solodynamo/go-init/master/goinit.sh
```

Comment:
Lets install the script at current location instead of home directory. Because of PFA
<img width="959" alt="screen shot 2019-02-26 at 2 50 16 pm" src="https://user-images.githubusercontent.com/17698714/53401604-25b64e80-39d6-11e9-8472-e8ea8ee42a3f.png">